.github/actions/install-python-and-package/action.yml

```diff
@@ -27,8 +27,10 @@ runs:
       with:
         update-conda: true
         python-version: ${{ inputs.python-version }}
-        conda-channels: anaconda
-    - run: conda --version
+        conda-channels: pytorch, pyg, bioconda, defaults, sbl, conda-forge
+    - run: |
+        conda --version
+        conda env list
```

Comment:
what's the point of this? Just to for inspection in case it fails?

Reply:
yes :)